nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/nuxeo-ai-bulk-add-button.html

```diff
@@ -53,6 +55,13 @@
               status: {
                   type: Object,
                   notify: true,
+              },
+
+              isAiEnabled: {
```

Comment:
nit: we've been using a method `_isAvailable()` in our actions, perhaps it'd be nice to follow the convention